src/SmoFacade/Server.cs

```diff
@@ -295,6 +295,29 @@ public void AddRole(LoginProperties login, RoleProperties role)
       matchingLogin.AddRole(role.Name);
     }
 
+    public User AddUser(UserProperties user, string database)
+    {
+      var db = Databases.SingleOrDefault(d => d.Name.Equals(database, StringComparison.InvariantCultureIgnoreCase));

```

Comment:
Can this re-use code here https://github.com/factset/AgDatabaseMove/blob/master/src/SmoFacade/Server.cs#L163 ?

Reply:
Yes it can!